filebeat/docs/modules/rabbitmq.asciidoc

```diff
@@ -0,0 +1,69 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-rabbitmq]]
+:modulename: rabbitmq
+:has-dashboards: false
+
+== RabbitMQ module
+
+This is the module for parsing https://www.rabbitmq.com/logging.html[RabbitMQ log files]
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+Parses https://www.rabbitmq.com/logging.html[single file format] introduced in 3.7.0.
+
+Tested with version 3.7.14 on MacOS 10.14.4.
```

Comment:
I think we can skip the MacOS version here as it's the pure log file we tests from 3.7.14.